### PR TITLE
Docupdating a1qa into main

### PR DESCRIPTION
**Pull request by MichaelHoepler**
Rebased doc branch. Should be able to merge now.

### CODE REVIEW

File: docs/DEVELOPMENT.md
Comment (simonwo) [inline docs/DEVELOPMENT.md:1]: Plz remove this and make it a proper page in the docs site.

File: docs/static/img/installation/bacalhau-docker-run1.png
Comment (simonwo) [inline docs/static/img/installation/bacalhau-docker-run1.png:1]: Please can we use text for these instead? Images are bad for accessibility and hard to edit when they go out of date.

**Review by simonwo — APPROVED** [review]
I can't really look through 150+ files, so just going to approve this given that all of the changes are in the docs folder.

**Comment by MichaelHoepler** [comment]
@wdbaruni Those are a lot of changes we did on the structure and some content of the documentation. If you have questions regarding the changes, please let me know.

File: docs/DEVELOPMENT.md
Comment (simonwo) [reply docs/DEVELOPMENT.md:1]: It's still here!

File: docs/static/img/installation/bacalhau-docker-run1.png
Comment (simonwo) [reply docs/static/img/installation/bacalhau-docker-run1.png:1]: Can I some kind of response to this request please?

**Comment by simonwo** [comment]
@MichaelHoepler there are still two comments from my previous review outstanding.

**Comment by MichaelHoepler** [comment]
Removed all the unnecessary files from the branch.

**Review by simonwo — APPROVED** [review]

**Pull request merged**